### ISSUE TITLE
HTML Cluster

### ISSUE THREAD

**Issue by radekdob** [maplibre/ngx-maplibre-gl#143]
Hi, i am using custom HTML for cluster.
I have noticed that created HTML tags are endlessly blinking while inspecting them via browser inspector.

The same applies to official example here https://maplibre.org/ngx-maplibre-gl/demo/ngx-cluster-html 

So I assume they are re-rendering or updating changing style, even if map is not moved etc.

I don't think that's is exptected behavior, also I suspect a lot of unnecessary ngZone runs.

**Comment by HarelM** [comment]
Can you clarify the problem? Not sure I fully understand it...

**Comment by radekdob** [comment]
Sorry for faulty issue. 
I had a lot of ngZone checks triggered from mgl-markers-for-clusters component, but the real root cause was [data] input of mgl-geojson-source which triggered changes.

tl;dr;
all is good, thanks ;)
